docs/tutorials/add-spatial-method.rst

```diff
@@ -63,6 +63,7 @@ Optionally, a new spatial method can also overwrite the default behaviour for th
 - :meth:`pybamm.SpatialMethod.broadcast`
 - :meth:`pybamm.SpatialMethod.mass_matrix`
 - :meth:`pybamm.SpatialMethod.process_binary_operators`
+- :meth:`pybamm.SpatialMethod.concatenation`
```

Comment:
Do we also need to add `node_to_edge` and `edge_to_node` to this tutorial? Can be done in another issue if we do.